README.md

```diff
@@ -1,22 +1,326 @@
-# PHP SDK Demo App
+# Optimizely PHP Tutorial
 
-This demo uses the PHP SDK, a part of Optimizely's Full Stack solution. Check out the [live demo](http://ec2-52-36-13-152.us-west-2.compute.amazonaws.com/).
+This tutorial enables you to quickly get started in your development efforts to create a PHP web application with the Optimizely X PHP SDK.
 
+The Optimizely PHP SDK provides core components to run full stack experiments anywhere in the code. The SDK handles aspects like: 
 
-## Optimizely Full Stack Overview
+- Bucketing: Used to designate users to a specific experiment variation
+- Conversion tracking
+- Reporting via Optimizely’s [Stats Engine](https://www.optimizely.com/statistics/)
 
-Optimizely Full Stack allows developers to run experiments anywhere in their code! The PHP SDK provides the core components to run a full stack experiment with Optimizely. It handles aspects like bucketing, which is used to designate users to a specific experiment variation, conversion tracking, and reporting via Optimizely’s [Stats Engine](https://www.optimizely.com/statistics/).  
+![images/screenshot.png](images/screenshot.png)
 
-* View the [PHP Getting Started Guide](http://developers.optimizely.com/server/getting-started/index.html?language=php)
+The Test App simulates an online retailer testing the effects of sorting products by price vs category. Run the app locally to mimic bucketing website visitors by entering unique user IDs into the search bar. For example:
 
-* View the reference [documentation](http://developers.optimizely.com/server/reference/index.html?language=php).
+* In the text input box, type the name “Joe” as the user ID. This simulate a unique visitor. 
+* Select a specific variation for "Joe". The variation that is given to "Joe" will be deterministic. Meaning, as long as the experiment conditions remain the same, "Joe" will always get the same variation.
```

Comment:
Starting the second sentence with `Meaning` seems odd as I say the sentence out loud. I think it can be reworded or merged with the previous sentence.